src/assignments/6-array.js

```diff
@@ -21,3 +30,21 @@ min=1
 
 */
 // start here writing the code for program2
+
+const task6b = () => {
+    let array = [1,4,2,3,6,8,32];
+
+    let [max, min] = array.reduce((max_min, current) => { 
+        // Maximum value 
+        if (current > max_min[0]) 
+           max_min[0] = current
+        // Minimum value 
+        if (current < max_min[1]) 
+           max_min[1] = current
+       return max_min; 
+     },[-Infinity, Infinity]) 
+    
+    alert(`Array: ${array}\n
+    max = ${max}\n
+    min = ${min}`);
+}
```

Comment:
It is quite complicated. 
Could be simpler: 
You can get the logic from here:
(https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php)